home/ivp-planning/machines/scale-objects.md

```diff
@@ -2,13 +2,7 @@
 
 The scaling objects offers a way to alter the size of every object. This is useful to make adjustments of objects that are not [customizable](customizable-machines.md).
 
-{% hint style="warning" %}
-Use the scaling function with care to avoid deformations or glitches. If it is possible to use a [customization ](customizable-machines.md)option of the object, always use the option.
-{% endhint %}
-
-{% hint style="info" %}
-Currently it is only possible to scale objects along the horizontal axis. A scaling along all axis is currently in development.
-{% endhint %}
+Exercise caution when using the scaling function to prevent distortions or errors. When opting for the [customization](customizable-machines.md) options of an object, keep in mind that scaling along the Z-axis is possible for some, it may not apply to most objects.
```

Comment:
This still does not communicate that using customizations is preferred over scaling.

```suggestion
Exercise caution when modifying scale to prevent distortions or errors. When changing an objects size via it's [customization](customizable-machines.md) options is possible, doing so should be preferred over changing it's scale.
```

The note about being able to scale along the Z-Axis should be a separate info hint and not in this warning. It has nothing to do with customization options.